src/Traits/ClientEndpointsTrait.php

```diff
@@ -1311,15 +1339,15 @@ public function ping(array $params = [])
 	 *
 	 * @param array{
 	 *     id: string, // (REQUIRED) Script ID
-	 *     context: string, //  Script context
-	 *     timeout: time, // Explicit operation timeout
-	 *     master_timeout: time, // Specify timeout for connection to master
-	 *     pretty: boolean, // Pretty format the returned JSON response. (DEFAULT: false)
-	 *     human: boolean, // Return human readable values for statistics. (DEFAULT: true)
-	 *     error_trace: boolean, // Include the stack trace of returned errors. (DEFAULT: false)
-	 *     source: string, // The URL-encoded request definition. Useful for libraries that do not accept a request body for non-POST requests.
-	 *     filter_path: list, // A comma-separated list of filters used to reduce the response.
-	 *     body: array, // (REQUIRED) The document
+	 *     context?: string, // Script context
+	 *     timeout?: int|string, // Explicit operation timeout
+	 *     master_timeout?: int|string, // Specify timeout for connection to master
+	 *     pretty?: bool, // Pretty format the returned JSON response. (DEFAULT: false)
+	 *     human?: bool, // Return human readable values for statistics. (DEFAULT: true)
+	 *     error_trace?: bool, // Include the stack trace of returned errors. (DEFAULT: false)
+	 *     source?: string, // The URL-encoded request definition. Useful for libraries that do not accept a request body for non-POST requests.
+	 *     filter_path?: string, // A comma-separated list of filters used to reduce the response.
+	 *     body: string|array, // (REQUIRED) The document. If body is a string must be a valid JSON.
```

Comment:
To get closer to passing PHPStan level 6 you might want to hint this as either `string|array<string, mixed>` or `string|array<mixed>` (if lists are allowed)